sygnal/apnspushkin.py

```diff
@@ -124,39 +130,87 @@ def __init__(self, name, sygnal, config):
         # without this, aioapns will retry every second forever.
         self.apns_client.pool.max_connection_attempts = 3
 
-    async def _dispatch_request(self, log, span, device, shaved_payload, prio):
+    def _map_event_dispatch_handler(self, n):
+        if n.event_id and not n.type:
+            return self._dispatch_event
+
+        if not self.event_handlers:
+            return self._dispatch_message
+
+        handler = self.event_handlers.get(n.type, None)
```

Comment:
```suggestion
        handler = self.event_handlers.get(n.type)
```

`None` is the default value of `default=`.